spring-boot-springdoc/src/main/java/com/baeldung/springdoc/repository/BookRepository.java

```diff
@@ -0,0 +1,29 @@
+package com.baeldung.springdoc.repository;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import org.springframework.stereotype.Repository;
+
+import com.baeldung.springdoc.model.Book;
+
+@Repository
+public class BookRepository {
+
+    private List<Book> books = new ArrayList<>();
```

Comment:
Obviously this is not the main point of the article, but I would probably use a Map instead of a List here.